packages/core/src/codewhispererChat/controllers/chat/chatRequest/converter.ts

```diff
@@ -119,6 +119,7 @@ export function triggerPayloadToChatRequest(triggerPayload: TriggerPayload): { c
                         additionalContext: triggerPayload.additionalContents,
                     },
                     userIntent: triggerPayload.userIntent,
+                    origin: 'IDE',
```

Comment:
lets remove this for now - this function is used by other features like context menu and inline chat